scripts/automation.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "Getting Python dependencies"
+pip install -r requirements.txt
```

Comment:
This will probably need to be `sudo -H pip install ....` and I would skip the requirements.txt and just put them inline with this command to make it easier to see all at once (I don't think this is going to be too large)